source/views/sis/student-work/detail.ios.js

```diff
@@ -103,6 +103,8 @@ function LastUpdated({when}: {when: string}) {
         Last updated:
         {' '}
         {moment(when, 'YYYY/MM/DD').calendar()}
+        {'\n'}
+        Powered by St. Olaf Student Employment job postings
```

Comment:
Could provide a link to https://www.stolaf.edu/apps/stuwork/index.cfm?fuseaction=SearchResults or https://www.stolaf.edu/apps/stuwork/

Reply:
See my comment on the android one.